engine/routes.py

```diff
@@ -119,6 +120,15 @@ def get_local_dir(input_data):
     return os.path.join(base_dir, request_dir)
 
 
+def get_final_dir(input_data):
+    base_dir = app.config['FINAL_ASYNC_DIR']
```

Comment:
is the intent to allow the configuration to be updated without restarting stream engine? I would expect this is a once and done thing, so `base_dir` should be set outside this function as a "const" variable. 

Setting the value once has the benefit of not reading from  disk every time this function is called.

Reply:
Moving LOCAL_ASYNC_DIR and FINAL_ASYNC_DIR config values to global constants. Yes, the app.config dictionary is only loaded once in /engine/__init__.py as far as I can see.